### DISCUSSION TITLE
Testing documentation

### DISCUSSION THREAD

**Discussion by csc530** [spectreconsole/spectre.console#1409] [Ideas]
I think there should be dedicated documentation for using the `Spectre.Console.Testing` classes on the main website as it's a little hidden that SpectreConsole supports testing with it. And it's not super easy looking through the [source code](https://github.com/spectreconsole/spectre.console/tree/main/test) test to figure out which classes to use and how for whatever purpose you have (as seen in #912).
Right now there is a blurb under best practices...

> Unit Testing Best Practices
For testing of console output, Spectre.Console has [IAnsiConsole](https://spectreconsole.net/api/Spectre.Console/IAnsiConsole) that can be injected into your application. The [Spectre.Console.Test](https://www.nuget.org/packages/Spectre.Console.Testing/) contains a set of utilities for capturing the output for verification, either manually or via a tool such as [Verify](https://github.com/VerifyTests/Verify).

I think it could be featured more prominently in its own section with use cases, examples, etc.  (I'm also happy to help start/contribute to these docs too).

**Comment by patriksvensson** [comment]
@csc530 Sounds like a very good idea. PR:s for this are more than welcome! 😄
